scripts/build-preinstalled-snap.js

```diff
@@ -26,19 +26,21 @@ function readFileContents(filePath) {
 const bundlePath = require.resolve('../dist/bundle.js');
 const iconPath = require.resolve('../images/icon.svg');
 const manifestPath = require.resolve('../snap.manifest.json');
+const typesPath = require.resolve('../types/preinstalled-snap.d.ts');
 
 // File Contents
 const bundle = readFileContents(bundlePath);
 const icon = readFileContents(iconPath);
 const manifest = readFileContents(manifestPath);
+const types = readFileContents(typesPath);
 
 const snapId =
-  /** @type {import('./build-preinstalled.snap').PreinstalledSnap['snapId']} */ (
+  /** @type {import('@metamask/snaps-controllers').PreinstalledSnap['snapId']} */ (
```

Comment:
Keeping this as a dev dependency since it is still useful for validating the type here.

However removing the definitions file - see related comment.